source/data_at_rest_encryption.rst

```diff
@@ -83,12 +83,21 @@ key and save it to a file by using the |openssl| command:
    $ openssl rand -base64 32 > mongodb-keyfile
 
 Then, as the owner of the ``mongod`` process, update the file permissions: only
-the owner should be able to read and modify this file:
+the owner should be able to read and modify this file. The effective permissions
+specified with the ``chmod`` command can either be **600** (only the owner may
+read and modify the file) or **400** (only the owner may read the file.)
 
 .. code-block:: bash
 
    $ chmod 600 mongodb-keyfile
 
+If ``mongod`` is started with the ``--relaxPermChecks`` option, the keyring file
+can be owned by ``root`` and ``mongod`` can read or change the file based on the
+group bit set accordingly. The effective keyring file permissions in this
```

Comment:
Please consider this variant:
```
If ``mongod`` is started with the ``--relaxPermChecks`` option and the keyring file
is owned by ``root`` then ``mongod`` can read the file based on the
group read bit set accordingly.
```

Reply:
This part: `mongod can read or change` should be `mongod can read` because:
1. mongod won't try to change key file
2. if group `write` bit is set mongod won't start

Reply:
Ready for review